idl/error.go

```diff
@@ -56,7 +56,7 @@ func (pe *ParseError) Error() string {
 	var buffer bytes.Buffer
 	buffer.WriteString("parse error\n")
 	for _, pe := range pe.Errors {
-		buffer.WriteString(fmt.Sprintf("  line %d: %s\n", pe.Pos.Line, pe.Err))
+		buffer.WriteString(fmt.Sprintf("  line %d:%d: %s\n", pe.Pos.Line, pe.Pos.Column, pe.Err))
```

Comment:
Consider implementing `String()` on `Position` to print `$line[:$column]`.

```
func (p Position) String() string {
  s := strconv.Itoa(p.Line)
  if c := p.Column; c > 0 {
    s += ":" + strconv.Itoa(c)
  }
  return s
}
```

Reply:
Funny. I had a note to do that by didn't come back to it over the weekend. I'll add that now.